src/Components/Projects/index.js

```diff
@@ -1,11 +1,61 @@
+import { useEffect, useState } from 'react';
+import Modal from './Modal';
 import styles from './projects.module.css';
 
-function Projects() {
+const Projects = () => {
+  const [projects, saveProjects] = useState([]);
+  const [selectedProject, saveSelection] = useState({});
+  const [showModal, saveShowModal] = useState(false);
+
+  useEffect(() => {
+    fetch(`${process.env.REACT_APP_API_URL}projects`)
+      .then((response) => response.json())
+      .then((response) => {
+        saveProjects(response.data);
+      });
+  }, []);
+
+  const deleteProject = async (id) => {
+    await fetch(`${process.env.REACT_APP_API_URL}projects/${id}`, {
```

Comment:
```suggestion
    await fetch(`${process.env.REACT_APP_API_URL}/projects/${id}`, {
```
REACT_APP_API_URL should not include the `/`.